pyrovision/datasets/openfire.py

```diff
@@ -4,195 +4,69 @@
 # See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
 
 from pathlib import Path
-import warnings
-import json
-from PIL import Image, ImageFile
+from torchvision.datasets import DatasetFolder
+from typing import Any, Callable, Optional
+from PIL import Image
+from .utils import download_dataset_from_url
 
-from torchvision.datasets import VisionDataset
-from .utils import download_url, download_urls
+__all__ = ['OpenFire']
 
-ImageFile.LOAD_TRUNCATED_IMAGES = True
 
-__all__ = ['OpenFire']
+url = "https://github.com/pyronear/pyro-vision/releases/download/v0.1.2/open_fire-dd0b5bb0.zip"
+url_sample = "https://github.com/pyronear/pyro-vision/releases/download/v0.1.2/open_fire_sample-06a07e48.zip"
+zip_filename = "open_fire.zip"
```

Comment:
Let's move those as class attributes of OpenFire :+1:

Reply:
Why? So the user of our dataset has to fill in a link that will never change

Reply:
I don't mean class constructor arguments, I do mean fixed class attributes :)

Reply:
In case we end up with different versions of OpenFire, there will be URLs for each of them, so it makes more sense to have those as class attributes than module variables I think